ecc/bls12381/g1.go

```diff
@@ -98,9 +98,12 @@ func (g G1) encodeBytes(compressed bool) []byte {
 		isBigYCoord = byte(g.y.IsNegative())
 	}
 
-	bytes := g.x.Bytes()
+	bytes, e := g.x.MarshalBinary()
+	err(e)
```

Comment:
Given that this is only used as part of the MarshalBinary, why can't we propagate the error? Obviously there never should be an error, but in that case bytes, _ would be clearer to me.

Reply:
updated now